### PR TITLE
update translate config

### LINKED ISSUE #19801: Add translate configs for 1.5, update unit tests
Add translate configs for 1.5 for Translator and ReverseTranslator.
Update wantVer to "1.5" in pkg/translate/translate_value_test.go.

https://github.com/istio/operator/blob/189faa14be794a283513507afd91dc842e50c961/pkg/translate/translate_value_test.go#L369
https://github.com/istio/operator/blob/189faa14be794a283513507afd91dc842e50c961/pkg/translate/translate_test.go#L225

### PR DESCRIPTION
**Pull request by richardwxn**
resolves: https://github.com/istio/istio/issues/19801 and https://github.com/istio/istio/issues/18392

### CODE REVIEW

**Review by elfinhe — COMMENTED** [review]
cool!

File: pkg/translate/translate_test.go [hunk 1]
```diff
@@ -221,8 +221,7 @@ func TestNewTranslator(t *testing.T) {
 		{
 			name:         "version 1.5",
```
Comment (elfinhe) [inline pkg/translate/translate_test.go]: please add one more test for 1.6, to make sure it will fallback to 1.5 instead of error.

Comment (richardwxn) [reply pkg/translate/translate_test.go]: done

**Review by elfinhe — APPROVED** [review]
LGTM

**Pull request merged**